tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -95,7 +102,27 @@ def parse_datafile(datafile, dfs):
         bulk_create_errors(unsaved_parser_errors, 1, flush=True)
         return errors
 
-    line_errors = parse_datafile_lines(datafile, dfs, program_type, section, is_encrypted, case_consistency_validator)
+    # Last resort to catch any un-caught exceptions during parsing and handle state appropriately
+    line_errors = {}
+    try:
+        line_errors = parse_datafile_lines(datafile, dfs, program_type, section,
+                                           is_encrypted, case_consistency_validator)
+    except Exception as e:
+        generate_error = util.make_generate_parser_error(datafile, None)
+        error = generate_error(schema=None,
+                               error_category=ParserErrorCategoryChoices.PRE_CHECK,
+                               error_message=("An unknown error occurred, and the file has been rejected. Please "
+                                              "contact the TDP Admin team at TANFData@acf.hhs.gov for further "
+                                              "assistance."),
```

Comment:
Hey @reitermb / @victoriaatraft, would you mind giving this message a once over since it will be STT facing? The context for the message is something went really bad while parsing (something we didnt expect) and we are rejecting the file. Since we don't have app alerting yet, I wanted this message to serve as a indicator that the STT should reach out to the admin team so they can investigate and inform dev if need be.

Reply:
hi @elipe17, if there's no way to specify the error that went wrong, then just a minor tweak to the messaging to be consistent with our emails:

```
"We're sorry, an unexpected error has occurred and the file has been rejected. Please"
"contact the TDP support team at TANFData@acf.hhs.gov for further"
"assistance."
```

Reply:
Thank you